src/components/Reviews-category.jsx

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useState } from "react";
+import { Link, useParams } from "react-router-dom";
+import { getReviewByCategory, getReviews } from "../api";
+import "../Reviews.css";
+
+const ReviewsCategory = ({ currCategory, setCurrCategory }) => {
+  const [categoryList, setCategoryList] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  let { category } = useParams();
+
+  useEffect(() => {
+    getReviews(category).then((reviews) => {
+      setCategoryList(reviews);
+      setIsLoading(false);
+    });
+  }, [category]);
+
+  return (
+    <section>
+      <h2 id="reviews-h2">Reviews for {currCategory} Games.</h2>
+      {isLoading ? (
+        <div>
+          <p id="loading-text">Loading Reviews for {currCategory} Games...</p>
+        </div>
+      ) : (
+        <section>
+          {categoryList.map((review) => {
+            return (
+              <article className="review-card" id="review-cat-card">
+                <Link
+                  to={`/reviews/${review.review_id}`}
+                  key={review.id}
```

Comment:
On line 33 where you are providing the key:
The unique key should be provided for each list element, here that would be the wrapping **article** tag. Also I think you might have meant review.review_id as review.id is most likely undefined and would still cause us errors 😱